main.go

```diff
@@ -38,9 +37,9 @@ func LoadConfigs(configName string) *PeacemakrConfig {
 		log.Printf("Error reading config, %v", err)
 		log.Println("Using default values instead")
 		configuration = PeacemakrConfig{
-				Verbose: true,
+				Verbose: false,
 				Host: "https://api.peacemakr.io",
-				PersisterFileLocation: "/tmp/",
+				PersisterFileLocation: "/tmp/.peacemakr",
```

Comment:
I am wondering, maybe this should be in the home directory of the current user? tmp sometimes gets blown away :)

Reply:
this is configurable in the yml file :)